lib/Backends/OpenCL/OpenCL.h

```diff
@@ -205,6 +205,13 @@ class OCLBackend final : public Backend {
     }
     return true;
   };
+
+  bool shouldLower(const Node *N) const override {
+    if (N->getKind() == Kinded::Kind::ConvolutionNodeKind)
```

Comment:
Please add a comment like: "OpenCL backend can handle any kind of convolutions, including group convolutions" or something like that.